### PR TITLE
Add button active and hover

### PR DESCRIPTION
**Pull request by karl-jerico**
- Add hover
- Add active button

### CODE REVIEW

File: src/components/DashboardMenu.tsx [hunk 1]
```diff
@@ -29,43 +29,64 @@ const DashboardMenu: React.FC = () => {
       <menu className="courses">
         <ul className="courses_list">
           <li>
-            <TabButton onSelect={() => handleSelect("main-course")}>
+            <TabButton
+              onSelect={() => handleSelect("main-course")}
+              active={selectedCourse === "main-course"}
+            >
```
Comment (mikco-hipe) [inline src/components/DashboardMenu.tsx:35]: create a variable for this "main-course" etc so that you dont have to write the string repeatedly

**Review by mikco-hipe — CHANGES_REQUESTED** [review]

File: src/styles/colors/_index.scss [hunk 2]
```diff
@@ -5,3 +5,4 @@ $button-quantity: #F2F0F0;
 $cart: #FDFBFC;
 $white: #FFFFFF;
 $sidebar: #7c7b7a;
+$dashboard-menu: #F9E3EB;
```
Comment (mikco-hipe) [inline src/styles/colors/_index.scss:8]: new line

**Review by mikco-hipe — CHANGES_REQUESTED** [review]

File: src/components/DashboardMenu.tsx (hunk 1 above)
Comment (Shiara-Deudor) [reply src/components/DashboardMenu.tsx:35]: (2)

**Review by Shiara-Deudor — CHANGES_REQUESTED** [review]

File: src/components/DashboardMenu.tsx [hunk 3]
```diff
@@ -1,101 +1,41 @@
...
-  Soup,
-  Dessert,
-  Bread,
-  Beverages,
-  Breakfast,
-  Snack,
-} from "@/utils/icons";
-
-import FoodApi from "@/components/foods/FoodApi";
+// DashboardMenu.tsx
```
Comment (mikco-hipe) [inline src/components/DashboardMenu.tsx:1]: remove comment

File: src/utils/courseData.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+// courseData.tsx
```
Comment (mikco-hipe) [inline src/utils/courseData.tsx:1]: remove comment

**Review by Shiara-Deudor — APPROVED** [review]
GEFM

**Review by mikco-hipe — APPROVED** [review]
🚢

**Pull request merged**